lms/urls.py

```diff
@@ -26,9 +26,6 @@
     url(r'^change_email$', 'student.views.change_email_request', name="change_email"),
     url(r'^email_confirm/(?P<key>[^/]*)$', 'student.views.confirm_email_change'),
     url(r'^change_name$', 'student.views.change_name_request', name="change_name"),
-    url(r'^accept_name_change$', 'student.views.accept_name_change'),
```

Comment:
@ormsbee I hear you don't want us to delete "old APIs". Is it OK to delete these name-change workflow APIs that we haven't used in a long, long time (and product doesn't want to save)?

Assuming you are OK with that, my next question is about deleting "change_name" itself, as that will happen through the new accounts API. Having to maintain "change_name" indefinitely as an independent API seems like a pain with questionable value.